requirements.txt

```diff
@@ -0,0 +1,3 @@
+ReFrame-HPC

```

Comment:
I don't think we need to have these and the README for running since we generally clone and run.

Reply:
yup, removed